app/pages/sections/order-items/catalogue-solutions/select/routes.js

```diff
@@ -84,17 +93,22 @@ export const catalogueSolutionsSelectRoutes = (authProvider, addContext, session
       const orderItems = sessionManager.getFromSession({ req, key: sessionKeys.orderItems });
       const alreadySelectedItem = orderItems
```

Comment:
I think `existingItem` might be a better name.

Reply:
changed to existingItem